src/main/java/org/jfrog/hudson/trigger/ArtifactoryTrigger.java

```diff
@@ -75,14 +77,20 @@ public void run() {
                         project.scheduleBuild(new ArtifactoryCause(itemLastModified.getUri()));
                     }
                 } else {
-                    logger.fine(job.getName() + " job received last modified time that is not newer for " + path);
+                    logger.fine(job.getName() + " job received last modified time (" + responseLastModified + ") that is not newer (" + lastModified +") for " + path);
```

Comment:
I suggest changing the above log line to:
**logger.fine(String.format("Artifactory trigger did not trigger job %s, since last modified time: %d is earlier than %d for path %s", job.getName(), responseLastModified, lastModified, path));**

Reply:
Done